SPDX-FileCopyrightText: 2021 Melg Eight <public.melg8@gmail.com>

SPDX-License-Identifier: GPL-3.0-or-later

Remove dependency on po folder because we don't have gettext yet.

--- ./Makefile.am
+++ ./Makefile.am
@@ -17,7 +17,7 @@
 
 ALL_RECURSIVE_TARGETS =
 
-SUBDIRS = po . gnulib-tests
+SUBDIRS = . gnulib-tests
 
 EXTRA_DIST =				\
   .mailmap				\
